doc/sphinx/Pacemaker_Explained/status.rst

```diff
@@ -1,280 +1,347 @@
+       .. index::
+          pair: lrm_rsc_op; transition-key
+
+       transition-key
+     - :ref:`text <text>`
+     - A concatenation of the action's transition graph action number, the
+       transition graph number, the action's expected result, and the UUID of
+       the controller instance that scheduled it.
+   * - .. _lrm_rsc_op_transition_magic:
+
+       .. index::
+          pair: lrm_rsc_op; transition-magic
+
+       transition-magic
+     - :ref:`text <text>`
+     - A concatenation of ``op-status``, ``rc-code``, and ``transition-key``.
+   * - .. _lrm_rsc_op_exit_reason:
+
+       .. index::
+          pair: lrm_rsc_op; exit-reason
+
+       exit-reason
+     - :ref:`text <text>`
+     - An error message (if available) from the resource agent or Pacemaker if
+       the action did not return success.
+   * - .. _lrm_rsc_op_on_node:
+
+       .. index::
+          pair: lrm_rsc_op; on_node
+
+       on_node
+     - :ref:`text <text>`
+     - The name of the node that executed the action (identical to the
+       ``uname`` of the enclosing ``node_state`` element)
+   * - .. _lrm_rsc_op_call_id:
+
+       .. index::
+          pair: lrm_rsc_op; call-id
+
+       call-id
+     - :ref:`integer <integer>`
+     - A node-specific counter used to determine the order in which actions
+       were executed.
+   * - .. _lrm_rsc_op_rc_code:
+
+       .. index::
+          pair: lrm_rsc_op; rc-code
+
+       rc-code
+     - :ref:`integer <integer>`
+     - The resource agent's exit status for this action. Refer to the *Resource
+       Agents* chapter of *Pacemaker Administration* for how these values are
+       interpreted.
+   * - .. _lrm_rsc_op_op_status:
+
+       .. index::
+          pair: lrm_rsc_op; op-status
+
+       op-status
+     - :ref:`integer <integer>`
+     - The execution status of this action. The meanings of these codes are
+       internal to Pacemaker.
+   * - .. _lrm_rsc_op_interval:
+
+       .. index::
+          pair: lrm_rsc_op; interval
+
+       interval
+     - :ref:`nonnegative integer <nonnegative_integer>`
+     - If the action is recurring, its frequency (in milliseconds), otherwise
+       0.
+   * - .. _lrm_rsc_op_last_rc_change:
+
+       .. index::
+          pair: lrm_rsc_op; last-rc-change
+
+       last-rc-change
+     - :ref:`epoch time <epoch_time>`
+     - Node-local time at which the action first returned the current value of
+       ``rc-code``.
+   * - .. _lrm_rsc_op_exec_time:
+
+       .. index::
+          pair: lrm_rsc_op; exec-time
+
+       exec-time
+     - :ref:`integer <integer>`
+     - Time (in seconds) that action execution took (if known)
+   * - .. _lrm_rsc_op_queue_time:
+
+       .. index::
+          pair: lrm_rsc_op; queue-time
+
+       queue-time
+     - :ref:`integer <integer>`
+     - Time (in seconds) that action was queued in the local executor (if known)
+   * - .. _lrm_rsc_op_op_digest:
+
+       .. index::
+          pair: lrm_rsc_op; op-digest
+
+       op-digest
```

Comment:
`op-restart-digest` and `op-secure-digest` are undocumented. Not necessary for this PR